### PR TITLE
VideoOptionsControlBar: Update toggle fullscreen button image

### PR DESCRIPTION
**Pull request by robwayne**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

**Comment by robwayne** [comment]
@carolanitz @Mikanbu ping!

File: Sources/VLCPlaybackController.m [hunk 1]
```diff
@@ -57,6 +57,9 @@ @interface VLCPlaybackController () <VLCMediaPlayerDelegate, VLCMediaDelegate, V
 
     NSUInteger _currentAspectRatio;
     BOOL _isInFillToScreen;
+    BOOL _toggledFullScreen;
+    NSUInteger _previousAspectRatio;
```
[the remark this answers is not in the document]
Comment (robwayne) [reply Sources/VLCPlaybackController.m]: Currently, it's necessary to accommodate the current (old) ui. The changeAspectRatio button toggles between different video ratios, but additionally you can enter directly into fullScreen by double tapping, therefore knowing when a request for fullScreen mode is requested is necessary

File: Sources/VLCMovieViewController.m [hunk 2]
```diff
@@ -1205,6 +1207,12 @@ - (void)hideShowAspectratioButton:(BOOL)hide
                      }];
 }
 
+- (void) playbackControllerDidSwitchAspectRatio:(NSUInteger)aspectRatio
+{
+    // if aspectRatio is VLCAspectRatioFillToScreen
+    _videoOptionsControlBar.isInFullScreen = aspectRatio == VLCAspectRatioFillToScreen;
```
Comment (carolanitz) [inline Sources/VLCMovieViewController.m]: you don't need that comment, that is already in the code

File: Sources/VideoOptionsControlBar.swift [hunk 3]
```diff
@@ -121,8 +128,6 @@ protocol VideoOptionsControlBarDelegate: class {
     
     override init(frame: CGRect) {
         repeatMode = .doNotRepeat
-        subtitleToggled = false
-        interfaceDisabled = false
```
Comment (carolanitz) [inline Sources/VideoOptionsControlBar.swift]: why was that removed as part of toggle fullscreen button image?

File: Sources/VideoOptionsControlBar.swift [hunk 4]
```diff
@@ -24,13 +24,20 @@ protocol VideoOptionsControlBarDelegate: class {
...
+    }
+    
+    var subtitleToggled: Bool = false {
         didSet {
             selectSubtitleButton.tintColor = subtitleToggled ? .orange : .white
         }
     }
     
-    var interfaceDisabled: Bool {
+    var interfaceDisabled: Bool = false {
```
Comment (carolanitz) [inline Sources/VideoOptionsControlBar.swift]: a bool is always already false when it's initialized

File: Sources/VLCPlaybackController.h [hunk 5]
```diff
@@ -42,6 +50,7 @@ currentMediaHasTrackToChooseFrom:(BOOL)currentMediaHasTrackToChooseFrom
 - (void)prepareForMediaPlayback:(VLCPlaybackController *)controller;
 - (void)showStatusMessage:(NSString *)statusMessage;
 - (void)displayMetadataForPlaybackController:(VLCPlaybackController *)controller metadata:(VLCMetaData *)metadata;
+- (void)playbackControllerDidSwitchAspectRatio:(NSUInteger)aspectRatio;
```
Comment (carolanitz) [inline Sources/VLCPlaybackController.h]: VLCAspectRatio not NSUInteger

File: Sources/VideoOptionsControlBar.swift (hunk 4 above)
Comment (robwayne) [reply Sources/VideoOptionsControlBar.swift]: Swift complains about initialization since its a property observer

File: Sources/VideoOptionsControlBar.swift (hunk 3 above)
Comment (robwayne) [reply Sources/VideoOptionsControlBar.swift]: Set their default values as false

File: Sources/VLCMovieViewController.m [hunk 6]
```diff
@@ -564,6 +564,8 @@ - (void)viewWillDisappear:(BOOL)animated
 
 - (void)viewDidDisappear:(BOOL)animated
 {
+    // reset toggleButton to default icon when movieviewcontroller is dismissed
+    _videoOptionsControlBar.isInFullScreen = NO;
```
Comment (carolanitz) [inline Sources/VLCMovieViewController.m]: that leads to a wrong state, if you go into fullscreen and switch to miniplayer and back to maxiplayer the button has the wrong state because of this

File: Sources/VLCPlaybackController.m [hunk 7]
```diff
@@ -834,6 +833,8 @@ - (void)switchAspectRatio
     if ([self.delegate respondsToSelector:@selector(showStatusMessage:)]) {
         [self.delegate showStatusMessage:[NSString stringWithFormat:NSLocalizedString(@"AR_CHANGED", nil), [self stringForAspectRatio:_currentAspectRatio]]];
     }
+
+    [_delegate playbackControllerDidSwitchAspectRatio:_currentAspectRatio];
```
Comment (carolanitz) [inline Sources/VLCPlaybackController.m]: you need to make a check here if the delegate responds to the selector, that call is under an @optional which means that a delegate doesn't need to implement it. Even though you might have a delegate this could crash when the function is not implemented

File: Sources/VLCMovieViewController.m (hunk 6 above)
Comment (carolanitz) [reply Sources/VLCMovieViewController.m]: I would set that in prepareForMediaPlayback which is only called when a new media is started

File: Sources/VLCPlaybackController.m (hunk 7 above)
Comment (robwayne) [reply Sources/VLCPlaybackController.m]: Okay, ahh thats what that does

File: Sources/VLCMovieViewController.m (hunk 6 above)
Comment (robwayne) [reply Sources/VLCMovieViewController.m]: Actually, prepareForMediaPlayback is called everytime the view comes back into view. But I didnt know that there was a playbackDidStop notification, which is better for resetting I believe.
